src/extract.ts

```diff
@@ -464,6 +497,55 @@ function extractCatch2Result(output: string): BenchmarkResult[] {
     return ret;
 }
 
+function extractJuliaBenchmarkHelper(g: JuliaBenchmarkGroup, labels: string[] = []): BenchmarkResult[] {
+    const res: BenchmarkResult[] = [];
+    for (const key in g[1].data) {
+        const value = g[1].data[key];
+        if (value[0] === 'BenchmarkGroup') {
+            for (const x of extractJuliaBenchmarkHelper(value as JuliaBenchmarkGroup, [...labels, key])) {
+                res.push(x);
+            }
+        } else if (value[0] === 'TrialEstimate') {
+            const v = value as JuliaBenchmarkTrialEstimate;
+            res.push({
+                name: [...labels, key].join('/'),
+                value: v[1].time,
+                unit: 'ns',
+                extra: `gctime=${v[1].gctime}\nmemory=${v[1].memory}\nallocs=${v[1].allocs}\nparams=${JSON.stringify(
+                    v[1].params[1],
+                )}`,
+            });
+        } else if (value[0] === 'Trial') {
+            throw new Error(
+                `Only TrialEstimate is supported currently. You need to apply apply an estimation (minimum/median/mean/maximum/std) before saving the JSON file.`,
+            );
+        } else {
+            throw new Error(`Unsupported type ${value[0]}`);
+        }
+    }
+    return res;
+}
+
+function extractJuliaBenchmarkResult(output: string): BenchmarkResult[] {
+    let json: JuliaBenchmarkJson;
+    try {
+        json = JSON.parse(output);
+    } catch (err) {
+        throw new Error(
+            `Output file for 'julia' must be JSON file generated by BenchmarkTools.save("output.json", suit::BenchmarkGroup) :  ${err.message}`,
+        );
+    }
+
+    const res: BenchmarkResult[] = [];
+    for (const group of json[1]) {
+        for (const r of extractJuliaBenchmarkHelper(group)) {
+            res.push(r);
+        }
```

Comment:
this can be done without looping
```suggestion
        res.push(...extractJuliaBenchmarkHelper(group));
```